client/src/Movies/SavedList.js

```diff
@@ -1,19 +1,24 @@
 import React, { Component } from 'react';
+import { NavLink, Link } from 'react-router-dom';
 
 export default class SavedList extends Component {
-  constructor(props) {
-    super(props);
-  }
+	constructor(props) {
+		super(props);
+	}
 
-  render() {
-    return (
-      <div className="saved-list">
-        <h3>Saved Movies:</h3>
-        {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
-        ))}
-        <div className="home-button">Home</div>
-      </div>
-    );
-  }
+	render() {
+		return (
+			<div className="saved-list">
+				<h3>Saved Movies:</h3>
+				{this.props.list.map(movie => (
+					<NavLink to={`/movies/${movie.id}`} className="saved-movie">
```

Comment:
This still works fine functionally, but React throws a warning in the console about putting a unique key prop on NavLink.

> index.js:2178 Warning: Each child in an array or iterator should have a unique "key" prop.

> Check the render method of `SavedList`. See https://fb.me/react-warning-keys for more information.
   > in NavLink (at SavedList.js:14)
   > in SavedList (at App.js:25)
   > in div (at App.js:24)
   > in App (at index.js:8)
   > in Router (created by BrowserRouter)
   > in BrowserRouter (at index.js:8)